lib/visual-website-optimizer/index.js

```diff
@@ -21,7 +31,19 @@ var VWO = module.exports = integration('Visual Website Optimizer')
  */
 
 VWO.prototype.initialize = function(){
-  if (this.options.replay) this.replay();
+   if (this.options.replay) {
+    var self = this;
```

Comment:
Move this outside of the `if` statement and remove the second `self` declaration. These are both scoped at that level anyway (JS does not have block scoping) so you're shadowing this variable